src/main/index.ts

```diff
@@ -146,9 +158,19 @@ if (!isFirstInstance) {
   };
 
   app.on('session-created', (session) => {
+    let reservedStateValues = [];
     if (!isDevelopment) {
       log.info('Cleaning local storage on session creation');
```

Comment:
You should update this log message to reflect that fact that you are not clearing the local storage in its entirety. Also, you should add additional logging lines below to log that backing up the local storage and restoring the local storage are completed respectively.